README.md

```diff
@@ -109,10 +110,23 @@ If you need to do some work on the container that's running the code, here's how
 docker exec -it mathesar_service bash
 ```
 
-To open a PostgreSQL psql terminal for the data in Mathesar:
-```
-docker exec -it mathesar_db psql -U mathesar
-```
+### Modifying data in the user database
+
+> ⚠️ **Note:** Mathesar caches schema information from the user database for up to 5 minutes. This cache will need to be invalidated before structural changes (e.g. adding a table) become visible within the UI. To invalidate this cache, change `DB_REFLECTION_INTERVAL` to `1` within `mathesar/reflection.py` and reload the UI (but don't commit this change).
```

Comment:
Actually this will be out of date once https://github.com/centerofci/mathesar/pull/1511 is merged.

Reply:
Oh fantastic. Ok I'll fix.

Reply:
Fixed in c47984746a9edd498f433ba74ec98cfe824bda09